src/main/java/org/folio/dew/error/BulkEditSkipListener.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.dew.error;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.des.domain.dto.JobCommand;
+import org.folio.dew.service.SaveErrorService;
+import org.springframework.batch.core.annotation.OnSkipInProcess;
+import org.springframework.batch.core.configuration.annotation.JobScope;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Log4j2
+@RequiredArgsConstructor
+@JobScope
+@Component
+public class BulkEditSkipListener {
+
+  private final SaveErrorService saveErrorService;
+
+  @Value("#{jobExecution.executionContext}")
+  private final ExecutionContext executionContext;
```

Comment:
You need a JobExecution here to get the file name parameter.
It should look like - jobExecution.getJobParameters.getString("fileName")

Reply:
Fixed, thank you. However, I used identifiersFileName instead of fileName.